lib/modules/versioning/node/readme.md

```diff
@@ -1,3 +1,9 @@
-Renovate's Node.js versioning is a wrapper around npm's versioning, except that it makes sure to strip "v" prefixes from exact versions when replacing.
+Renovate's Node.js versioning is a wrapper around npm versioning.
+But Renovate removes any `v` prefixes from semantic versions when replacing.
 
-It is planned to extend it one day to support "stability" awareness, because Node.js's version stability does not follow the SemVer approach.
+Its primary purpose is to add Node.js LTS awareness, e.g.:
+
+- Odd releases are unstable
+- Even releases do not reach stability (LTS) immediately
+
+Unfortunately, there is not currently any way to get `node` versioning to replace `docker` versioning when Docker image tags contain a compatibility suffix such as `-alpine` or `-slim`, because npm versioning treats these suffixes as implying pre-releases.
```

Comment:
This sentence is too long, and has too many conditions in it.

Can we chop things up like this, maybe? We should also say upfront _who_ is the targe audience for the paragraph.

```suggestion
If you use Docker, read this paragraph.
You can _not_ use `node` versioning to replace `docker` versioning.
This is because npm versioning treats these suffixes as implying pre-releases.
Using `node` versioning would helped you when Docker image tags have a compatibility suffix like `-alpine` or `-slim`.
```